cpp/tests/utilities/column_utilities.cu

```diff
@@ -69,5 +118,61 @@ void expect_equal_buffers(void const* lhs, void const* rhs,
                             typed_rhs));
 }
 
+struct column_view_printer {
+  template <typename Element, typename std::enable_if_t<is_numeric<Element>()>* = nullptr>
+  void operator()(cudf::column_view const& col, const char *delimiter, std::ostream &ostream) {
+
+    cudf::size_type num_rows = col.size();
+    std::vector<Element> h_data(num_rows);
+    CUDA_TRY(cudaMemcpy(h_data.data(), col.data<void *>(),
+                        num_rows * sizeof(Element),
+                        cudaMemcpyDeviceToHost));
+
+    if (col.nullable()) {
+      cudf::size_type null_size = (num_rows - 1 + cudf::detail::size_in_bits<bitmask_type>()) /
+        cudf::detail::size_in_bits<bitmask_type>();
+
+      std::vector<bitmask_type> h_null(null_size);
+      CUDA_TRY(cudaMemcpy(h_null.data(), col.null_mask(),
+                          cudf::detail::size_in_bits<bitmask_type>() * null_size,
```

Comment:
Is this correct? 32 * null_size bytes copied? Shouldn't it be just `sizeof(bitmask_type) * null_size`? 

Better yet, can you just use `thrust::host_vector<bitmask_type> h_null(col.null_mask())` to automatically copy it to host?

Reply:
Yes, bad computation.  I'll rework this as I look at to_host